tests/functional/test_utilities_batch.py

```diff
@@ -117,6 +119,17 @@ def handler(record):
     return handler
 
 
+@pytest.fixture(scope="module")
+def sqs_fifo_record_handler() -> Callable:
+    def handler(record):
+        body = record["body"]
+        if "fail" in body:
+            raise Exception("Failed to process record.")
+        return body
+
+    return handler
```

Comment:
naive question, do we need it given it's the exact same logic as `record_handler` fixture?

Reply:
Absolutely right. I created it in the beginning because I thought it would be different, but it's not :) Changing it